core/settings.py

```diff
@@ -412,6 +412,7 @@ def expose_site_context(request):
 ADRESSE_DATA_GOUV_SEARCH_TYPE_HOUSENUMBER = "housenumber"
 ADRESSE_DATA_GOUV_SEARCH_TYPE_STREET = "street"
 IN_MUNICIPALITY_SEARCH_TYPE = "in_municipality"
+IN_DEPARTEMENT_SEARCH_TYPE = "departement"
```

Comment:
Inconsistent naming between this line and the above one.
(`in_` prefix + FR/EN)